modules/Oculus_module/include/#HandRetargeting.hpp#

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
`.hpp#` ?

Reply:
File autogenerated by emacs. Removed in https://github.com/robotology/walking-teleoperation/pull/8/commits/5be53b54d032767ef445450249062c4fae0eee04